src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -131,6 +158,10 @@ public function merge(AttributeMetadataInterface $attributeMetadata)
         if (null === $this->serializedName) {
             $this->serializedName = $attributeMetadata->getSerializedName();
         }
+
+        foreach (\array_keys($attributeMetadata->getEmbedProperties()) as $property) {
```

Comment:
IMHO no point for calling `array_keys` instead of:
```suggestion
        foreach ($attributeMetadata->getEmbedProperties() as $property => $value) {
```

Reply:
I used it to avoid having a non used $value :sweat_smile: